src/devices/Pn5180/SelectedPiccInformation.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Device.Rfid;
+
+namespace Iot.Device.Pn5180
+{
+    internal class SelectedPiccInformation
+    {
+        public Data106kbpsTypeB Card { get; set; }
+        public bool LastBlockMark { get; set; }
```

Comment:
I'm having hard time figuring out what does this mean and how it is PiccInformation, should this actually belong here?

Reply:
It's related to the 14443-4 communication protocol. You have a bit changing which is called the Block Mark. And you have to adjust when you write or get a value to read.

Reply:
fair enough, thanks for clarifying!